src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE users(
+    id bigint GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    first_name varchar(50) NOT NULL,
```

Comment:
Sorry, I missed that at the first review. Why `first_name` is `NOT NULL`? According to requirements, only `email` and `password` are required

Reply:
Alright, I will change it to nullable, but I just thought that a user without a username would be kinda strange.

Reply:
I do agree with you but on the other side, we will have the easiest sign-up process by just entering email and password. Personally speaking, I don't like filling a lot of fields during the registration process that's why I prefer to abide by the requirements to implement this User API.
What do you think about that? Do you think it would be better to have first and last names as required fields?

Reply:
You can leave `created_at` as `NON NULL`